src/bisection.jl

```diff
@@ -31,6 +31,11 @@ function SciMLBase.solve(prob::IntervalNonlinearProblem, alg::Bisection, args...
             retcode = ReturnCode.ExactSolutionLeft, left = left,
             right = right)
     end
+    if iszero(fr)
```

Comment:
Also, i dont see `RightExactSolution` condition in any of these bracketing solvers, was that intentional? Otherwise, it's gonna calculate an inexact solution even if the right value is the exact zero?

Reply:
If there is a condition by which right solution is exact it should do that retcode yes.